tutorials/beginner/configuring.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: tutorial-beginner
+title: Configuring Functionality
+---
+
+{% include base.html %}
+
+# Working with rules and scripts
+
+**This part is not yet written.**
+
+## Prerequisites
+
+* Installed [openHab Designer](http://docs.openhab.org/installation/designer.html) and read/write access to the **CONF** folder of your openHAB server
```

Comment:
"Eclipse SmartHome Designer"

**CONF** is not in sync with the rest of the documentation. The conf folder my be called different. For example, in a repository based installation on linux, your conf folder is `/etc/openhab2`. That's why it will be better to write "to your openHAB configuration folder"